pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -485,6 +541,45 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Set routers for a particular domain.
+		#[pallet::weight(T::WeightInfo::set_domain_multi_router())]
+		#[pallet::call_index(9)]
+		pub fn set_domain_multi_router(
+			origin: OriginFor<T>,
+			domain: Domain,
+			routers: BoundedVec<T::Router, T::MaxRouterCount>,
+		) -> DispatchResult {
+			T::AdminOrigin::ensure_origin(origin)?;
+
+			ensure!(domain != Domain::Centrifuge, Error::<T>::DomainNotSupported);
+			ensure!(routers.len() > 0, Error::<T>::InvalidMultiRouter);
```

Comment:
Why not `== T::MaxRouterCount`? Later in the incoming message, we do not need to check for the router counts. We need that it's always well configured.

Reply:
That's better, I'm gonna change this.